### PR TITLE
feat: added tutorial to profile dialog and refactor

### PR DESCRIPTION
**Pull request by Emmanuel-Develops**

### CODE REVIEW

**Comment by Extheoisah** [comment]
I think it's redundant to have the hamburger for opening the sidebar as well as the profile separately. We can have the profile and signout links inside the drawer/sidebar, remove the hamburger and use just the profile image as the button to open the drawer/sidebar. Hence, the sidebar will show -> pages, profile, signout. A similar design is used by github (see attached images)
Screenshot 2023-06-26 at 15 46 09
Screenshot 2023-06-26 at 15 49 36

**Comment by Emmanuel-Develops** [comment]
> I think it's redundant to have the hamburger for opening the sidebar as well as the profile image which opens a popover separately. We can combine both and have the profile and signout links inside the drawer/sidebar, remove the hamburger and use just the profile image as the button to open the drawer/sidebar. Hence, the sidebar will show -> pages, profile, signout. A similar design is used by github (see attached images) Screenshot 2023-06-26 at 15 46 09 Screenshot 2023-06-26 at 15 49 36

True, Jonas said the same. This PR however removes the sidebar and includes the routes in the profile dialog

I think the github implementation is better in the long run, should we change it to that now or focus on other issues?

**Comment by Extheoisah** [comment]
> > I think it's redundant to have the hamburger for opening the sidebar as well as the profile image which opens a popover separately. We can combine both and have the profile and signout links inside the drawer/sidebar, remove the hamburger and use just the profile image as the button to open the drawer/sidebar. Hence, the sidebar will show -> pages, profile, signout. A similar design is used by github (see attached images) Screenshot 2023-06-26 at 15 46 09 Screenshot 2023-06-26 at 15 49 36
> 
> True, Jonas said the same. This PR however removes the sidebar (with hamburger) and includes the routes in the profile dialog
> 
> I think the github implementation is better in the long run, should we change it to that now or focus on other issues?

makes sense to switch it once and for all

File: src/components/navbar/Menu.tsx [hunk 1]
```diff
@@ -14,81 +14,147 @@ import {
...
+                      <MenuNav
+                        currentRoute={currentRoute}
+                        routeName="home"
+                        routeLink={ROUTES_CONFIG.HOME}
+                        handleClose={closeMenu}
+                        icon={FiHome}
+                      />
+                      {userSession?.user?.githubUsername && (
+                        <MenuNav
+                          routeName="profile"
```
Comment (Extheoisah) [inline src/components/navbar/Menu.tsx:125]: why isn't this in the `ROUTES.CONFIG` file?

File: src/components/navbar/MenuNav.tsx [hunk 2]
```diff
@@ -22,21 +22,27 @@ const MenuNav = ({
...
+            color={isCurrentRoute ? "orange.200" : "gray.600"}
             as={icon}
           />
         )}
         <Text
-          alignSelf="flex-end"
-          _groupHover={isCurrentRoute ? {} : { textDecoration: "underline" }}
+          textAlign="right"
+          flex="1 1 auto"
+          // ml="auto"
```
Comment (Extheoisah) [inline src/components/navbar/MenuNav.tsx:41]: any reason for keeping this commented code?

**Review by Extheoisah — COMMENTED** [review]
Looks good. Just a few clean up comments

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**